File: IOS/Sarja.swift
//
//  Sarja.swift
//  uusinJakso
//
//  Created by jukka tuomi on 18.1.2017.
//  Copyright © 2017 jukka tuomi. All rights reserved.
//

import UIKit

class Sarja: NSObject {
    var nimi: String
    var uusinJakso: String
    
    init(fromNimi sarjanNimi:String){
        nimi = sarjanNimi
        uusinJakso = ""
        
    }
    func asetaJakso(fromJakso jakso: String){
        uusinJakso = jakso
    }
    func toString() -> String {
        return nimi + ": " + uusinJakso
    }
}

File: Android/Sarja.java
package com.example.uusinjakso;


public class Sarja {
    private String nimi;
    private String uusinJakso;


    public Sarja(String nimi,String uusinJakso){
        this.nimi = nimi;
        this.uusinJakso = uusinJakso;
    }
    public String haeJakso(){
        return this.uusinJakso;
    }
    @Override
    public String toString(){
        return this.nimi + "  uusin jakso: " + this.uusinJakso;
    }


}